app/models/voxelytics/VoxelyticsDAO.scala

```diff
@@ -134,10 +137,24 @@ class VoxelyticsDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContex
           a.inodeCount,
           a.version,
           a.metadata,
-          t.name AS taskName
+          t.name AS taskName,
+          o.workflow_hash AS other_workflow_hash,
+          o._run AS other_run
         FROM webknossos.voxelytics_artifacts a
         JOIN latest_complete_tasks t ON t._id = a._task
-        """.as[(String, String, String, String, Long, Long, String, String, String)])
+        LEFT JOIN ( -- when the task is skipped, the artifact from the producing task run is joined for linking
```

Comment:
please add the discussed distinct constraint to the schema, and a comment here explaining how the fanout effect is prevented (joining at most one item, due to the distinct constraint)